pymc4/inference/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pymc4 import _backend
+
+if _backend.TENSORFLOW:
```

Comment:
I thought we were ditching the idea of multiple backends?

Reply:
Not actually, we ditch the idea to put additional abstractions for now and treat every backend separately. So far we have the only one. Once we come up with a new backend we may try to implement it.